src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -593,7 +593,7 @@ private function addAnnotationsSection(ArrayNodeDefinition $rootNode)
             ->children()
                 ->arrayNode('annotations')
                     ->info('annotation configuration')
-                    ->addDefaultsIfNotSet()
+                    ->canBeDisabled()
```

Comment:
Looks like we are already facing this. If I install framework-bundle + console directly and create a simple console application ontop of this - it should immediately fail as annotation not installed and the default configuration is to enable them.

Since these are the defaults - it is really not obviously that I have annotations enabled somewhere.